packages/lab/src/combo-box-next/ComboBoxNext.css

```diff
@@ -0,0 +1,13 @@
+.saltComboBoxNext-highlight {
+  font-weight: var(--salt-text-fontWeight-strong);
+}
+
+.saltComboBoxNext-input:hover {
+  --input-borderColor: var(--salt-focused-outlineColor);
```

Comment:
since we're outside the component, i think this would usually be --saltInput-borderColor (does that token exist?)

Reply:
I don't think we do